src/quicktype-graphql-input/index.ts

```diff
@@ -160,6 +160,7 @@ function expandSelectionSet(selectionSet: SelectionSetNode, inType: GQLType, opt
 interface GQLSchema {
     readonly types: { [name: string]: GQLType };
     readonly queryType: GQLType;
+    readonly mutationType?: GQLType;
```

Comment:
I don't think it matters here, but `?` has different semantics than `X | undefined`.  `X | undefined` means the property must be present but its value can be `undefined`.  `?` means the property can be missing.

Reply:
I think "missing" is more appropriate in this case anyway, since a GraphQL endpoint without any mutation operations wouldn't have a `mutationType`.